### DISCUSSION TITLE
Provide updated version outside docker

### DISCUSSION THREAD

**Discussion by DorianCoding** [dani-garcia/vaultwarden#5220] [Ideas]
Hello,

It seems only docker/podman allows to have automatic updates of the package. Else it needs to be installed again by recompiling or pulled from docker.
Is there any on going work on this like an external repo, system-wide packages... ?

Thanks.

**Comment by BlackDex** [comment]
The official way we support currently is docker. 
The community has some other ways https://github.com/dani-garcia/vaultwarden/wiki/Third-party-packages

But those are not maintained by this project.

**Reply by DorianCoding** [reply]
> The official way we support currently is docker. The community has some other ways https://github.com/dani-garcia/vaultwarden/wiki/Third-party-packages
> 
> But those are not maintained by this project.

And there is no "official" other way that are planned to be implemented? Or maybe just promoting some community efficient and reliable ways?

**Reply by BlackDex** [reply]
No there is no other way planned.
The wiki link i provided has several options for different platforms. Depending on your OS some have official packages in there repo like Alpine and Arch Linux.
